libs/island-ui/core/src/lib/BreadcrumbsDeprecated/Breadcrumbs.tsx

```diff
@@ -0,0 +1,38 @@
+/** @deprecated BreadcrumbsDeprecated has been deprecated in favor of BreadCrumbs */
+import React, { FC, Children } from 'react'
+import { Text } from '../Text/Text'
+import { Icon } from '../Icon/Icon'
+import { Colors } from '@island.is/island-ui/theme'
+import * as styles from './Breadcrumbs.treat'
+
+interface BreadcrumbsProps {
+  label?: string
+  color?: Colors
+  separatorColor?: Colors
+}
+
+export const Breadcrumbs: FC<BreadcrumbsProps> = ({
+  label = 'breadcrumb',
+  color = 'blue400',
+  separatorColor = 'blue400',
+  children,
+}) => {
+  const crumbs = Children.toArray(children).filter((c) => c)
+
```

Comment:
we should probably add the deprecated hook to this component if it is deprecated